plugins/RemoteControl/src/RequestHandler.hpp

```diff
@@ -69,6 +69,18 @@ public slots:
 	//! @warning Make sure to only call this only when the server is offline because they are not synchronized
 	void setPassword(const QString& pw);
 
+	//! Sets wether CORS is enabled.
+	//! @warning Make sure to only call this only when the server is offline because they are not synchronized
+	void setEnableCors(bool v);
+	//! Returns if CORS is enabled
+	//! @warning Make sure to only call this only when the server is offline because they are not synchronized
+	bool getEnableCors() { return enableCors; }
+	//! @warning Make sure to only call this only when the server is offline because they are not synchronized
+	void setCorsHosts(const QString& hosts);

```

Comment:
please document format of the hosts string, including separators if really more than one host is allowed.

Reply:
I just documented this to clarify it's either one host or "*" (any host). More logic is needed if we are to support multiple specific hosts, because we need to adjust the response header to match the requester origin in case it's whitelisted. I think this can wait until it's needed, right?

Reply:
Yes. My short investigation what this all means indicated that no current browser understands more than one address. You may reflect this by using singular in method and argument names. Sorry to be so picky, but it improves understandability.

Reply:
Oh, I see what you mean. I never noticed that because "*" is a possible value which means "all hosts", that's why plural made sense to me. But I'll change it to singular if you prefer, no problem.

Reply:
Agree :)

Reply:
Done. I also took the opportunity to rename "host" to "origin", as it's the proper technical term.